### ISSUE TITLE
Make {{>example-fixture}} work with more than 1 example

### ISSUE THREAD

**Issue by bjedrzejewski** [ScottLogic/d3fc#703]
We often use {{>example-fixture}} to simplify the website code. However, when more than one example is there, we need to use something like:

```
<div id="series_point_2" class="chart"> </div>
<script type="text/javascript">
(function() {
    var desiredWidth = $('#series_point_2').width(),
        desiredHeight = desiredWidth / 2.4; //keeps the width-height ratio at 600-250 (defaults for createFixture)
    var f = createFixture('#series_point_2', desiredWidth, desiredHeight, null, function() { return true; });
    var container = f.container, data = f.data,
      xScale = f.xScale, yScale = f.yScale;
    {{{example-code-2}}}
}());
</script>
```

Which is a bit disappointing. Solution for this problem would be great.

**Comment by jdunkerley** [comment]
If I ever get the playground to work will look at these

**Comment by chrisprice** [comment]
It would probably start with burning assemble but I would also love to see
this

**Comment by ColinEberhardt** [comment]
> However, when more than one example is there, we need to use something like:

I have to confess ... I am responsible for this nasty cut'n'paste approach. I blame Assemble. Burn it.

**Comment by chrisprice** [comment]
Fixed by 2bf4206de72876ee4b4044f2ae88f691c54400eb
